facilitator/src/aggregation.rs

```diff
@@ -265,15 +289,34 @@ impl<'a> BatchAggregator<'a> {
                     ingestion_packet.uuid));
             }
 
-            if !server
-                .aggregate(
+            let mut did_aggregate_shares = false;
+            let mut last_err = None;
+            for server in servers.iter_mut() {
+                match server.aggregate(
                     &ingestion_packet.encrypted_payload,
                     &VerificationMessage::try_from(peer_validation_packet)?,
                     &VerificationMessage::try_from(own_validation_packet)?,
-                )
-                .context("failed to validate packets")?
-            {
-                invalid_uuids.push(peer_validation_packet.uuid);
+                ) {
+                    Ok(valid) => {
+                        if !valid {
+                            invalid_uuids.push(peer_validation_packet.uuid);
+                        }
+                    }
+                    Err(e) => {
+                        last_err = Some(Err(e));
+                        continue;
+                    }
+                }
+                did_aggregate_shares = true;
```

Comment:
I think this would read more clearly if you moved `did_aggregate_shares = true` and `break` into the `Ok(valid)` branch since they will only execute in that case.